tests/router/common.py

```
@@ -2175,13 +2202,61 @@ async def test_sync():
+
+        def _create_router():
+            nonlocal kv_router, kv_router_error
+            try:
+                with min_initial_workers_env(expected_num_instances):
+                    kv_router = KvRouter(
+                        endpoint=endpoint,
+                        block_size=block_size,
+                        kv_router_config=kv_router_config,
+                        aic_perf_config=aic_perf_config,
+                    )
+            except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)